Karamelfile

```diff
@@ -20,8 +20,13 @@ dependencies:
       - kube-hops::node
       - kube-hops::master
       - hops::docker_registry
-# This is needed for 1.3
-      - kube-hops::post_conf
```

Comment:
Yes it can be removed. It was on master because that's where Karamel picks up the Karamelfile, no matter which branch you are installing. Hence the comment.